### PR TITLE
create first spring classes

### PR DESCRIPTION
**Pull request by MarianBastiurea**

### CODE REVIEW

File: .DS_Store
Comment (andreiursudev) [inline .DS_Store:1]: Dont commit .DS_Store files

File: src/main/java/spring_framework/App01Gaming.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package spring_framework;
```
Comment (andreiursudev) [inline src/main/java/spring_framework/App01Gaming.java]: Use proper package name like `com.marianbastiurea.learnspringframework`.
Organize the project in chapters like you did in `java-fun` project

File: notes [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/*
```
Comment (andreiursudev) [inline notes]: Rename file to `notes.txt`.

**Review by andreiursudev — CHANGES_REQUESTED** [review]

File: notes (hunk 2 above)
Comment (MarianBastiurea) [reply notes]: done

File: src/main/java/spring_framework/App01Gaming.java (hunk 1 above)
Comment (MarianBastiurea) [reply src/main/java/spring_framework/App01Gaming.java]: I tried when I started this project but start.spring.io didn't want to let me to do that

Comment (MarianBastiurea) [reply src/main/java/spring_framework/App01Gaming.java]: create new packages

File: .DS_Store
Comment (MarianBastiurea) [reply .DS_Store:1]: done

File: src/main/java/spring_framework/App01Gaming.java (hunk 1 above)
Comment (MarianBastiurea) [reply src/main/java/spring_framework/App01Gaming.java]: done

**Review by andreiursudev — APPROVED** [review]

**Pull request merged**